package.json

```diff
@@ -20,13 +21,19 @@
   "author": "Vijay Krishna Bammidi",
   "license": "ISC",
   "dependencies": {
+    "@babel/plugin-transform-runtime": "^7.9.0",
+    "@babel/preset-env": "^7.9.0",
+    "@babel/runtime": "^7.9.2",
     "@hapi/joi": "^16.1.8",
+    "acorn": "^7.1.1",
     "babel": "^6.23.0",
     "babel-core": "^6.26.3",
     "body-parser": "^1.19.0",
     "csv-parser": "^2.3.2",
     "csvtojson": "^2.0.10",
+    "dotenv": "^8.2.0",
     "eslint": "^6.7.2",
+    "jest": "^25.1.0",
```

Comment:
A small note - don't forget that it's better to keep testing dependencies in "devDependencies" section so that they won't affect the bundle size of your production experience.

Reply:
Added to the dev dependencies